src/main/resources/application.conf

```diff
@@ -15,8 +15,8 @@ no.nav.security.jwt {
     issuers = [
         {
             issuer_name = ${?OIDC_ISSUER}
```

Comment:
Kan denne miljøvariabelen droppes? Mulig den blir utledet fra discovery-url-en?

Reply:
Hm, virker som tokensupport forventer denne variablen, for appen vil ikke starte hvis jeg tar den bort, men det virker ikke som den bryr seg om verdien den blir satt til. Jeg prøvde å sette den til `abc` og da fungerer appen som forventet.... 🙈 Hva tenker du jeg burde gjøre?

Reply:
Det virker som `issuer_name` er et kallenavn i ktor versjonen til tokensupport. I spring-boot-støtten så brukes den til å referere til en eksplisitt issuer basert på `issuer_name` dersom man kun ønsker tokens fra denne på et gitt endepunkt med en annotasjon, ellers så kan man godta tokens fra flere issuers. I Ktor så har de ikke like god granularitet. Vi kan definere flere issuers i konfigurasjonen, men alle endepunktene med denne konfigurasjonen vil da godta tokens fra alle issuerne. I det tilfellet er issuer_name bare et kallenavn eller en referanse som gjør det lett for oss å skille mellom issuerne på. Mange setter bare `selvbetjening` eller `loginservice` eller noe slikt der.